SwiftificationTests/SequenceTypeTests.swift

```diff
@@ -39,4 +39,24 @@ class SequenceTypeTests: XCTestCase {
         XCTAssertEqual(expected, actual)
     }
     
-}
+    func testUnique() {
+        let array = [1, 1, 2, 2, 3, 4]
+        let unique = array.unique()
+        XCTAssertTrue(array.count == 6)
+        XCTAssertTrue(unique.count == 4)
```

Comment:
I would test that `unique` is equal to `[1, 2, 3, 4]` since I think preserving order is important.